### DISCUSSION TITLE
How to customize “back to Article” header in a Manuscript site?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#10389] [Q&A]
I want to customize the header of the Notebook view inside a manuscript. Are there partials for that? I imagined manuscript was an extension but I couldn't find its files so I imagine it is something g baked into Quarto itself. Where is the source code? Can I create my own manuscript template with custom html and typst  (.typ) ?

**Comment by mcanouil** [comment]
Look at the language: https://quarto.org/docs/authoring/language.html
